src/Processors/Formats/Impl/BinaryRowInputFormat.h

```diff
@@ -54,7 +56,7 @@ class BinaryWithNamesAndTypesSchemaReader : public FormatWithNamesAndTypesSchema
     BinaryWithNamesAndTypesSchemaReader(ReadBuffer & in_, const FormatSettings & format_settings_);
 
 private:
-    BinaryFormatReader reader;
+    BinaryFormatReader<false> reader;
```

Comment:
How is it compiling if the templates are not explicitly instantiated in cpp file? 👀